packages/code-du-travail-data/indexing/addGlossary.js

```diff
@@ -0,0 +1,56 @@
+import { v4 as uuidv4 } from "uuid";
+const glossary = require("@socialgouv/datafiller-data/data/glossary.json");
+
+const conventionMatchers = [
+  "convention collective",
+  "conventions collectives",
+  "accords de branches",
+  "accord de branche",
+  "disposition conventionnelle",
+  "dispositions conventionnelles",
+];
+
+export function addGlossary(htmlContent) {
+  const internalRefMap = new Map();
+  const formatedContent = glossary.reduce((updatedContent, match) => {
+    const { abbrs, title, definition, variants } = match;
+    if (!updatedContent) return updatedContent;
```

Comment:
Ok je vois ! Est-ce que ce serait pas un peu plus intuitif ça ?
```suggestion
export function addGlossary(htmlContent = []) {
  
  if (!htmlContent.length) return htmlContent; // ou juste un tableau vide ou meme null !
  const internalRefMap = new Map();
  const formatedContent = glossary.reduce((updatedContent, match) => {
    const { abbrs, title, definition, variants } = match;
    if (!updatedContent) return updatedContent;
```